src/translate.rs

```diff
@@ -61,26 +61,22 @@ pub trait Ptr: Copy {
 }
 
 impl <T> Ptr for *const T {
-    fn is_null(&self) -> bool {
-        (*self).is_null()
-    }
+    #[inline]
+    fn is_null(&self) -> bool { (*self).is_null() }
```

Comment:
Not sure that was super useful to reduce to one line but I guess it doesn't really matter...